<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelstr1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>str1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_str2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>str2</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>212</width>
      <height>205</height>
     </rect>
    </property>
    <property name="title">
     <string>字符串</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="bntAppend">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bntPrepend">
       <property name="text">
        <string>Prepend</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="bntToUpper">
       <property name="text">
        <string>toUpper</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="bntToLower">
       <property name="text">
        <string>toLower</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="bntLeft">
       <property name="text">
        <string>left</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="bntRight">
       <property name="text">
        <string>right</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bntSection">
       <property name="text">
        <string>section</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="bntSimplified">
       <property name="text">
        <string>simplified</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="bntTrimed">
       <property name="text">
        <string>trimmed</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>117</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>数字</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="bntCount">
       <property name="text">
        <string>count</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="bntIndexof">
       <property name="text">
        <string>indeOf</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="bntSize">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="bntLasteIndex">
       <property name="text">
        <string>lastIndexof</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>逻辑判断</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bntContains">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>contains</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntIsNull">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>isNull</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntIsempty">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>97</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>isEmpty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntEndswith">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>endswith</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntStartWith">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>startsWith</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>结果</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>265</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkCotains">
       <property name="text">
        <string>isNull</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>contains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox1_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>520</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>G:\Qt5Book\QT5.9Study</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G:\Qt5Book\QT5.9Study\qw.cpp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学生姓名，男，1984-3-4，汉族，山东</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox2_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>520</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>\</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.cpp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>:</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>，</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edtResult_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>520</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
